src/mui/input/CheckboxGroupInput.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component, PropTypes } from 'react';
+import Checkbox from 'material-ui/Checkbox';
+import Labeled from './Labeled';
+
+/**
+ * An Input component for a checkbox group, using an array of objects for the options
+ *
+ * Pass possible options as an array of objects in the 'choices' attribute.
+ *
+ * By default, the options are built from:
+ *  - the 'id' property as the option value,
+ *  - the 'name' property an the option text
+ * @example
+ * const choices = [
+ *    { id: 'M', name: 'Male' },
```

Comment:
Bad example, there is no interest for a checkbox here as gender cannot be both "male" and "female" (not that I'm aware of).

Same for following examples